content/learn/migration-guides/0.16-to-0.17.md

```diff
@@ -6,7 +6,21 @@ weight = 12
 long_title = "Migration Guide: 0.16 to 0.17"
 +++
 
-The most important changes to be aware of this release are:
+The most important breaking changes to be aware of this release are:
+
+- `Event` has been split into `Message` (for buffered events) and `Event` (for observers)
+- various methods, types and calling patterns around observers have been renamed or slightly tweaked
+- many rendering related types have been moved to new crates as we've attempted to improve modularity, code organization and compile times
+- web builds now require additional boilerplate in the form of a special `RUSTFLAGS` environment variable due to upstream changes in `get_random`
```

Comment:
Maybe link the CLI alpha here?

Reply:
Already in the detailed migration guide for this change!